docker/jaxrstck/README

```diff
@@ -1,8 +1,10 @@
 The pom.xml in this folder can be used to run the Jakarta REST Standalone TCK 
-against Glassfish 6.1.0+ (with Java SE 11+ compatability)
+against Glassfish 6.1.0+ (with Java SE 11+ compatability) 
+also using https://github.com/jakartaee/rest/tree/master/jersey-tck
+
 
 Below are the instructions to run the Jakarta REST TCK that is built from 
```

Comment:
FYI for others reading this pull request, the files in [docker/jaxrstck](https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/docker/jaxrstck) are not distributed (currently) with any TCKs, instead these notes and maven pom is for verifying that the REST TCK passes with GlassFish.  IMO, `pom.xml` for other compatible implementations besides GlassFish could be added in [this folder](https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/docker/jaxrstck) as well (if other RESTful Web Service implementations choose to create a pull request for doing so before EE 10 goes final).  

CC @brideck @jamezp ^